app/components/post_list/post/post.tsx

```diff
@@ -297,6 +297,12 @@ const Post = ({
             <CallsCustomMessage
                 serverUrl={serverUrl}
                 post={post}
+
+                // Note: the below are provided by the index, but typescript seems to be having problems.
```

Comment:
Here's the type error when those are removed: 
```
Type '{ serverUrl: string; post: PostModel; }' is missing the following properties from type 'Omit<Omit<Props, "isMilitaryTime" | "currentUser"> & OwnProps & WithDatabaseArgs, "database">': isAdmin, otherParticipants, isHost, joiningChannelId
```
So it appears that it's not realizing that the props are being set in the index. I'm not sure why it's having trouble here, but not in the other places we're doing exactly the same thing. I've even tried to specify the props in the index (instead of spreading the props) and it still can't figure it out. 
Open to better solutions of course, I just couldn't fix it.

Reply:
Heh, forgot to ping you for the above @enahum . :)